docs/guis/multiturn/excitation.md

```diff
@@ -0,0 +1,207 @@
+
+??? info "Kick Amplitudes at Injection"
+
+    As the beams are not particularly hard at injection, small kick amplitudes lead to large peak to peak oscillations and we generally use small amplitudes.
+    A reasonable starting point is anywhere between __1%__ and __3%__, then going up slowly in steps of __2%__, until beam losses during kicks stop being reasonable.
+
+??? info "Kick Amplitudes in the Ramp"
+
+    Performing kicks in the ramp requires careful planning.
+    As the beam energy increases, so does the beam rigidity harder and hence larger kick amplitudes can be used.
+    Nevertheless, careful monitoring of losses during acquisitions and adjusting the kick amplitudes accordingly is crucial.
+
+    Typically, we prepare a table various kicks to be performed, indicating the time in the ramp, corresponding energy, phase knob setting, ATS factor, kick amplitude and optics file.
+    These should follow the various match points for the given energy ramp program.
+    <!-- TODO: check this first! Most of these information can be found by searching `LSA` in the `CCM`, then navigating to and opening `Generator`, `Edit` mode and searching & selecting the relevant beam process. -->
+
+    An __example table__ is shown below.
+
+    | Time  | Energy (TeV) | Phase Knob | ATS  | Kick Amplitude (%) |   Optics    |
+    |:-----:|:------------:|:----------:|:----:|:------------------:|:-----------:|
+    |  30s  |     0.46     |    100%    |  1   |         3          | TODO: write |
+    | 240s  |     1.0      |    50%     |  1   |         7          | TODO: write |
+    | 405s  |     1.9      |     0%     |  1   |         13         | TODO: write |
+    | 580s  |     2.9      |     0%     |  1   |         19         | TODO: write |
+    | 720s  |     3.7      |     0%     |  1   |         24         | TODO: write |
+    | 860s  |     4.5      |     0%     |  1   |         30         | TODO: write |
+    | 1010s |     5.5      |     0%     | 0.75 |         36         | TODO: write |
+    | 1160s |     6.2      |     0%     | 0.57 |         41         | TODO: write |
+    | 1247s |     6.6      |     0%     | 0.5  |         45         | TODO: write |
+
+    The values in this table are a good starting point, but it is important to monitor the losses and reduce the kick amplitudes accordingly.
+
+??? info "Kick Amplitudes at Top Energy"
+
+    At top energy the increased beam rigidity allows us larger kick amplitudes.
+    A reasonable starting point is __5%__, then going up in steps of __5%__ until beam losses during kicks stop being reasonable.
+    Remember that it is very time-consuming to get back to this state when losing the beam at top energy, so monitor beam losses carefully and be reasonable with the kick increases.
+
+### Exciting the Beam
+
+Trigger an acquisition by clicking the yellow ++"Acquire with ACDipole excitation"++ button at the bottom left of the GUI.
```

Comment:
Add
```css
.yellow-gui-button kbd {
  background-color: rgb(241, 226, 83) !important;
  color: black !important;
  box-shadow: 0 .1rem 0 .05rem var(--md-typeset-kbd-border-color),0 .1rem 0 var(--md-typeset-kbd-border-color),0 -.1rem .2rem rgb(192, 182, 127) inset;;
}
``` 
to the `custom_colors.css` and `{.yellow-gui_button}` after the  second `++`. Then you can remove "yellow" in the text.

Reply:
I don't like adding CSS for every single button color